dogu.json

```diff
@@ -1,6 +1,6 @@
 {
   "Name": "official/plantuml",
-  "Version": "2023.10-5",
+  "Version": "2024.7-1",
```

Comment:
We don't need to update the version yet, because bumping the version is part of the later release process.

Reply:
Okay, reverted the version change